.github/workflows/deploy.yml

```
@@ -0,0 +1,36 @@
+name: deploy
+
+on:
+  workflow_run:
+    workflows: [documentation.yml]
+    types: [completed]
+    branches: [main]
+
+permissions:
+  contents: read
+
+jobs:
```

```suggestion
jobs:
 on-success:
```

I think we need this to ensure that we only deploy if build of doxygen and readthedocs was successful

---

I agree that we might need such a check but isn't `on-success` just the name of the job?

---

true, I read too fast over the docs

---

Condition added.